CHANGELOG.md

```diff
@@ -9,10 +9,15 @@ Web UI version numbers should always match the corresponding version of LBRY App
 ## [Unreleased]
 ### Added
   * Add setting to automatically purchase low-cost content without a confirmation dialog
+  * New custom styled scrollbar (#574)
```

Comment:
I like overall style and that it doesn't overlap header, but can it be wider and/or get wider on hover?

Reply:
Ok, increased the size, if users start to report some problems with this 
I'll make it more wider, but I think it's ok for now